src/theme/colour-primitive/specs/pa-colour.ts

```diff
@@ -0,0 +1,95 @@
+import { PrimitiveColourSet } from "../../types";
+
+export const PAColourSet: PrimitiveColourSet = {
+    "brand-10": "#0E123A",
+    "brand-20": "#191E65",
+    "brand-30": "#232B90",
+    "brand-40": "#303BC5",
+    "brand-50": "#494CFF",
+    "brand-60": "#AD6EFF",
+    "brand-70": "#AEA4FF",
+    "brand-80": "#C5C4FF",
+    "brand-90": "#DDDCFF",
+    "brand-95": "#EDECFF",
+    "brand-100": "#F8F8FF",
+    "primary-10": "#FCF2F3",
```

Comment:
side note... with primary being reddish, I guess everything will look like this?

![Screenshot 2025-03-03 at 6 06 52 PM](https://github.com/user-attachments/assets/a25b442e-e113-458e-adfc-5ec4a9e86c93)

Reply:
checking with UX but we will probably figure out if colour edits are needed as we go....?

Reply:
the current text-primary fails the WCAG contrast check, unfortunately https://webaim.org/resources/contrastchecker/?fcolor=DC666F&bcolor=FFFFFF

anyway feel free to tweak along the way